jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -408,4 +411,63 @@
               ./hack/jenkins/e2e.sh
             fi
     jobs:
-        - 'kubernetes-pull-{suffix}'
+        - '{git-project}-pull-{suffix}'
+
+- project:
+    name: node-pull
+    git-project:
+        - 'cadvisor':
+            repo-name: 'google/cadvisor'
+            git-basedir: 'github.com/google/cadvisor'
+            suffix: 'build-test-e2e'
+            owner: 'stclair@google.com'
+            max-total: 1
+            trigger-phrase: 'test'
+            status-context: 'Jenkins GCE e2e'
+            cmd: |
+                cd go/src/github.com/google/cadvisor
+                go get -u github.com/tools/godep
+                ./build/presubmit.sh
+
+                godep go build -tags test github.com/google/cadvisor/integration/runner
+                ./runner --logtostderr --test-retry-count=8 --test-retry-whitelist=integration/runner/retrywhitelist.txt \
+                --ssh-options "-i /var/lib/jenkins/gce_keys/google_compute_engine -o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o CheckHostIP=no -o StrictHostKeyChecking=no" \
+                e2e-cadvisor-ubuntu-trusty \
+                e2e-cadvisor-container-vm-v20151215 \
+                e2e-cadvisor-container-vm-v20160127 \
+                e2e-cadvisor-rhel-7
+
+                # Temporarily disabled for #1344
+                # e2e-cadvisor-coreos-beta \
+
+                # Old Images:
+                #e2e-cadvisor-ubuntu-trusty-docker19 \
+                #e2e-cadvisor-coreos-beta \
+                #e2e-cadvisor-rhel-7-docker111 \
+                #e2e-cadvisor-container-vm-v20160321
```

Comment:
Does anyone know why this image is disabled? Either way, it shouldn't be in the "old images" section...